src/contracts/AludelFactory.sol

```diff
@@ -232,23 +227,13 @@ contract AludelFactory is Ownable {
         delete _programs[program];
     }
 
-    /// @notice retrieves the full list of templates
-    /// @dev template values is an unbounded array
-    function getTemplates()
-        external
-        view
-        returns (EnumerableSet.TemplateData[] memory)
-    {
-        return _templates.values();
-    }
-
```

Comment:
At some point I was hesitant to remove this but now we can use the subgraph to list all our templates